Minecraft_Quick_Homes/Minecraft_Quick_Homes.ahk

```diff
@@ -427,58 +865,105 @@ Fix_Virtual_Shift_Held()
     return
 }
 
+toggle_auto_switch_sections()
+{
+    
+    IniRead, Auto_Switch_Sections_by_Minecraft_Title, HomeStorage.ini, config, Auto_Switch_Sections_by_Minecraft_Title
+
+    switch Auto_Switch_Sections_by_Minecraft_Title
+    {
+        case 0:
+            IniWrite, 1, HomeStorage.ini, config, Auto_Switch_Sections_by_Minecraft_Title
+            ending_toggle_message := "ON For Auto_Switch_Sections_by_Minecraft_Title Toggled ON"
+        goto exit_switch
+
+        case 1:
+            IniWrite, 0, HomeStorage.ini, config, Auto_Switch_Sections_by_Minecraft_Title
+            ending_toggle_message := "OFF For Auto_Switch_Sections_by_Minecraft_Title Toggled OFF"
+        goto exit_switch
+
+        Default:
+            ; If the user needs to teleport away, but Auto_Switch_Sections_by_Minecraft_Title was invalid after program was launched, set it to 0. 
```

Comment:
Might need to be changed to 1. The docs in issue #42 does need to be updated in case this changes. 

Asking the players for this use case would be wise. 
I think I might want to switch it to 1.